acme/acme/client.py

```diff
@@ -32,11 +32,8 @@
 # for SSL, which does allow these options to be configured.
 # https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning
 if sys.version_info < (2, 7, 9):  # pragma: no cover
-    try:
-        requests.packages.urllib3.contrib.pyopenssl.inject_into_urllib3()  # type: ignore
-    except AttributeError:
-        import urllib3.contrib.pyopenssl  # pylint: disable=import-error
-        urllib3.contrib.pyopenssl.inject_into_urllib3()
+    import urllib3.contrib.pyopenssl  # pylint: disable=import-error
```

Comment:
Do we need a heavier assurance that `urllib3` will be properly available now?

Reply:
Can't leave it as is in master -- the reason `requests` now has dependencies is that it removed its vendorized packages. The line as it is in master errors with the version of `requests` that we pin.

Reply:
ah. right. that's what it was. lint complains:
```
E: 36, 8: Module 'requests.packages' has no 'urllib3' member (no-member)
```
Technically true, but only in theory, not in practice. You comfortable with me squelching the error?

Reply:
I think so! The attribute exists but `lint` doesn't understand the code that magically adds it.

Do you have concerns about doing it?

Reply:
Nope, good enough for me.